File: SPL231-Assignment3-student-template/server/src/main/java/bgu/spl/net/impl/stomp/Frame.java
package bgu.spl.net.impl.stomp;
import java.util.HashMap;
public class Frame {

    private String command;
    private HashMap<String, String> headers;
    private String body;
    public Frame(String command, HashMap<String, String> headers, String body){
        this.command = command;
        this.headers = headers;
        this.body = body;
    }
    public String getCommand() {
        return command;
    }
    public HashMap<String, String> getHeaders() {
        return headers;
    }
    public String getBody() {
        return body;
    }
}

File: SPL231-Assignment3-student-template/client/src/Frame.cpp
#include "../include/Frame.h"
#include <string>
#include <unordered_map>
using namespace std;


Frame::Frame(string command, unordered_map<string,string> headers, string body): command(command),
    headers(headers),body(body){}

string Frame::getCommand(){
    return command;
};
unordered_map <string,string> Frame::getHeaders(){
    return headers;
};
string Frame::getBody(){
    return body;
};